src/docs/teams/debugger/debugging-a-transaction.md

```diff
@@ -29,4 +29,8 @@ Assuming your contract(s) have received one or more transactions you will now se
   <figcaption class="text-center">The Truffle Teams Transaction Monitoring screen.</figcaption>
 </figure>
 
-The debugger can be started by clicking on the **<span class="inline-button"><i class="fas fa-debug"></i> DEBUG</span>** button located on the right-hand-side of a given transaction.
+<p class="alert alert-info">
+<strong>Note</strong>: The Truffle Teams Debugger is currently only available for transactions excuted sent to contracts in <a href="/docs/teams/sandboxes/sandboxes-overview">Ganache sandboxes</a> and accordingly the DEBUG button will show as disabled for other networks. Debugging transactions made against public networks will be available in a forthcoming release.
+</p>
+
```

Comment:
you also need to add something about "if you don't see the button at all, you need to enable early access"

Reply:
👌 done

Reply:
wrestled with the wording a little so tweaks totally welcome